proto/vmexec.proto

```diff
@@ -46,9 +46,18 @@ message InitializeResponse {
   // This page intentionally left empty.
 }
 
+message UnmountWorkspaceRequest {}
```

Comment:
Can you add brief docs for these APIs?

Reply:
+1

Reply:
Done, and moved the service to the top of the file so that the docs come first.